packages/atomic/src/components/result-template-components/result-template-decorators.ts

```diff
@@ -7,7 +7,10 @@ export function ResultContext() {
       const element = getElement(this);
       const parentResultComponent = (element.getRootNode() as ShadowRoot)
         .host as HTMLAtomicResultElement;
-      if (!parentResultComponent) {
+      if (
+        !parentResultComponent ||
+        parentResultComponent.nodeName !== 'ATOMIC-RESULT'
```

Comment:
sneaking this in here cause the other problem reminded me we weren't checking for the type. Cypress tests for results & templates will be worked on soon

Reply:
Is there a "better way" to extract the nodeName from a component ? 

To know that `atomic-result` is named exactly like that, versus a soft check like this ?

Reply:
Atomic-result is a primitive in a Component decorator, putting variables as values of decorator options is not possible, so we have to write it out by hand